notebooks/book2/29/kf_parallel.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Write %%capture in first line (before comment) to avoid output of instllation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>